services-camel/src/main/java/edu/unc/lib/dl/services/camel/triplesReindexing/TriplesReindexingRouter.java

```diff
@@ -0,0 +1,82 @@
+/**
+ * Copyright 2008 The University of North Carolina at Chapel Hill
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package edu.unc.lib.dl.services.camel.triplesReindexing;
+
+import static org.apache.camel.LoggingLevel.INFO;
+import static org.fcrepo.camel.FcrepoHeaders.FCREPO_URI;
+
+import org.apache.camel.BeanInject;
+import org.apache.camel.LoggingLevel;
+import org.apache.camel.builder.RouteBuilder;
+
+/**
+ * Route for processing reindexing requests.
+ *
+ * @author bbpennel
+ *
+ */
+public class TriplesReindexingRouter extends RouteBuilder {
+
+    private static final String LDP_CONTAINS = "<http://www.w3.org/ns/ldp#contains>";
+
+    @BeanInject(value = "indexingMessageProcessor")
+    private IndexingMessageProcessor indexingMessageProcessor;
+
+    @Override
+    public void configure() throws Exception {
+        onException(Exception.class)
+            .redeliveryDelay("{{error.retryDelay}}")
+            .maximumRedeliveries("{{error.maxRedeliveries}}")
+            .backOffMultiplier(2)
```

Comment:
would it make sense to capture the backoff multiplier value in a property as well?

Reply:
Added